projects/vdk-plugins/vdk-duckdb/src/vdk/plugin/duckdb/duckdb_connection.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import pathlib
+import tempfile
+from typing import List
+
+import duckdb
+from vdk.internal.util.decorators import closing_noexcept_on_close
+
+log = logging.getLogger(__name__)
+
+
+class DuckDBConnection:
+    """
+    Create file based DuckDB database.
+    """
+
+    def __init__(
+        self,
+        duckdb_file: pathlib.Path = pathlib.Path(tempfile.gettempdir()).joinpath(
+            "vdk-duckdb.db"
```

Comment:
Shouldn't we be able to pass the filename somehow? What if my db is located somewhere else, like `~/duck.db`?